### DISCUSSION TITLE
What is the sync behaviour regarding conflicts?

### DISCUSSION THREAD

**Discussion by DDzwiedziu** [osmandapp/OsmAnd#20206] [Q&A]
I have several devices syncing and I'm running into a situation where I'm trying to sync two of them. Now last time I've did it the duplication bug wasn't fixed and I've ended up with everything triplicated (and even twice that when at some point I've started creating more categories and moving POIs from the default one), with no simple way to `sort | uniq` them. Which is not something I wanted to be worried about when syncing.

And now I'm not sure if both the devices have POIs that are not on the counterpart.
What will happen if I resolve the conflict? Because simple wording of ours/theirs per categories and the duplicating bug fixed leaves me with thinking that data will be lost at the device that “loses” the ours/theirs choice.

Edit: I think #18247 answers my question the way I did not want…

**Comment by vshcherb** [comment]
If you have OSM Edits on both and sync via OsmAnd cloud than it's a bad idea cause they will get probably duplicated on upload. If you talk about favorites than they resolve conflicts by name and location
